java/developer-guide/working-with-microsoft-graph-client/how-to-use-graphclient-for-microsoft-graph/_index.md

```diff
@@ -0,0 +1,290 @@
+---
+title: How to use GraphClient for Microsoft Graph
+type: docs
+weight: 20
+url: /java/how-to-use-graphclient-for-microsoft-graph/
+---
+
+
+## **Working with [GraphClient](https://apireference.aspose.com/email/java/com.aspose.email/GraphClient)**
```

Comment:
Remove the link from the header. The tree of content on the page breaks if using links in headers.